README.md

```diff
@@ -130,6 +130,18 @@ $stream->name; // string with the name
 $stream->layoutClassList; // array with the layout class list
 ```
 
+You can get information about all the streams in a session by calling the `listStreams($sessionId)` method of the
+`OpenTok\OpenTok` class.
+
+```php
+use OpenTok\Session;
+
+// Get list of streams from just a sessionId (fetched from a database)
+$streamList = $opentok->listStreams($sessionId);
+
+$streamList->totalCount(); // total count
```

Comment:
The readme for other SDK does not document all exposed methods

Reply:
Good point, will add them for other methods in a separate PR